src/Html.php

```diff
@@ -6997,11 +6999,11 @@ public static function timestampToRelativeStr($ts)
             if ($day_diff == 1) {
                 return __('Yesterday');
             }
-            if ($day_diff < 7) {
+            if ($day_diff < 14) {
                 return sprintf(__('%s days ago'), $day_diff);
             }
             if ($day_diff < 31) {
```

Comment:
Maybe we could add `last week` case like we have the `next` week case for future dates.
Also checks on future dates seems to be more precise. Maybe they could be reused for past dates.
Then, some strings are starting with an uppercase letter, but not other, it should probably be harmonized.